### ISSUE TITLE
[A11y] General - user agents cannot override styles in many cases

### ISSUE THREAD

**Issue by jechols** [OregonDigital/OD2#1813]
### Operating System

Windows

### Browser

Edge

### Affected page

Homepage

### Admin user?

Yes

### Issue Description

Testing out https://www.w3.org/WAI/WCAG21/Understanding/text-spacing.html, it seems we have a lot of places where text spacing fixes will not apply.

I'm seeing a whole lot of rules ending in `!important` in our CSS, and some of that is likely causing this problem, but I can't nail down exactly which rules mean what without more in-depth investigation.  Generally, though, we shouldn't use `!important` anywhere, simply because of the problems for user-defined overrides like this.

Here's an example of the problem in the header.  Before applying the test transform:

![image](https://user-images.githubusercontent.com/2157974/128372574-092b94a1-3e90-4ebd-adde-5235c14654ab.png)

After:

![image](https://user-images.githubusercontent.com/2157974/128372637-8ae0eb65-e1e0-487d-a66f-c023dabb40bb.png)

For reference, here's an example of successful text transformation:

Before:

![image](https://user-images.githubusercontent.com/2157974/128372090-46a98ecf-0108-4f5f-ba3f-8c98ea2fdfc6.png)

After:

![image](https://user-images.githubusercontent.com/2157974/128372154-e3c183cd-1ae6-4356-915b-d6977fb98403.png)

### Suggested Fix

If possible remove all `!important` rules and replace them with either more specific CSS (so the CSS can properly override less specific rules) or some similar fix.  Otherwise, if fixing those isn't feasible, it may be a massive project to simply figure out which `!important` rules are blocking the three critical overrides (character spacing, word spacing, and line height) across *the entire application*.

**Comment by jechols** [comment]
I saw this same problem in the collection landing page (#1838), and concluded that in that case the spurious uses of `!important` were not to blame.  May be that this is similar; I'm not sure because when looking at this one I found myself lost in the weeds of CSS hell and had to make an educated guess about the problem.
